pkg/utils/testUtils/kubernetes.go

```diff
@@ -1,4 +1,4 @@
-package util
+package testutils
```

Comment:
Would change the folder name to `testutils` for consistency. Could also be `testutil` to avoid having to rename the references below.